src/test/java/teammates/test/cases/action/BaseActionTest.java

```diff
@@ -131,7 +131,7 @@ protected ImageResult getImageResult(Action a) {
                 Const.ParamsNames.STUDENT_PROFILE_EMAIL, "e@email.com  ",
                 Const.ParamsNames.STUDENT_PROFILE_INSTITUTION, " TEAMMATES Test Institute 5   ",
                 Const.ParamsNames.STUDENT_NATIONALITY, "American",
-                Const.ParamsNames.STUDENT_GENDER, "  other   ",
+                Const.ParamsNames.STUDENT_GENDER, "OTHER",
```

Comment:
Should keep this to check what happens if there are trailing spaces, invalid values, etc.

Reply:
@wkurniawan07 sir Here Obviously if "OTHER" will have trailing space would give the error but why should we use unnecessary test cases if this would never happen in normal condition As Gender would be selected by the Radio Button

Reply:
Such a test is not unnecessary - POST parameters can easily be modified by the user.

Reply:
@whipermr5  Sir this issue has been solved with the introduction of if- else condition in ui/controller/studentprofileEditSaveAction I had used the concept as suggested by @wkurniawan07 
if not MALE/FEMALE then OTHER

Reply:
@shaharyar-shamshi, this test still needs to be there, as in the way it was before. While the handling of invalid values is indeed implemented that way, you still need to test it.

Reply:
@wkurniawan07  Sir Made the change with " OTHER "  still there